n2y/plugins/mermaid.py

```diff
@@ -64,8 +65,14 @@ def to_pandoc(self):
                 '-o', temp_filepath,
             ], capture_output=True, input=diagram_as_bytes, check=True)
             with open(temp_filepath, 'rb') as temp_file:
-                content = temp_file.read(4096)
-                url = self.client.save_file(content, self.page, '.png')
+                content = temp_file.read()
+                root = root = Path(__file__).resolve().parent.parent
+                with open(root/'data'/'mermaid_err.png', 'rb') as err_img:
+                    if content == err_img.read():
```

Comment:
Hey @sHermanGriffiths I'm confused as to what's happening here. Can you explain it a bit? Is this error image something the mermaid cli creates by default when there is a syntax error and we're checking to see if the bytes match to see if an error was thrown? 

I haven't looked into this but I'd definitely see if there is another way to handle this since checking for an entire image seems a bit odd.

Reply:
That's exactly right. Since mermaid doesn't throw an error, the only way to know there was an error is if this image is rendered.